### DISCUSSION TITLE
Export #REQ_BIRTHDATE to CSV or print

### DISCUSSION THREAD

**Discussion by ko873** [liedekef/events-made-easy#650] [Q&A]
Hi, how can I export the data from #REQ_BIRTHDATE to CSV or display it at the print page?
I can see the data from #REQ_BIRTHDATE when editing a booking but can´t display it anywhere else.

**Comment by liedekef** [comment]
Currently the custom people fields are not shown in the bookings overview.
The next version will include the people ID, so you can export the people and the bookings and link them in excel as you see fit.
However: underneath the event name in the booking overview, you have an "export to csv" link, and that link includes all booking and people info. So that's the way you want your info.

**Comment by ko873** [comment]
Thanks for the quick reply!
Unfortunately, #REQ_BIRTHDAY is not displayed in the CSV export that I can do in the list of events. Do I need to adjust the template for the export?

**Comment by liedekef** [comment]
O sorry, I was thinking about custom fields. You're correct, the birthday-info is not exported there. But this change will help:
https://github.com/liedekef/events-made-easy/commit/2d3c6b34caa5e95e6e327c3d430df7f848cf2732
